assets/js/components/PresenterStreamArea.tsx

```diff
@@ -12,12 +14,59 @@ type PresenterStreamAreaProps = {
 export const presenterStreams: { [key: string]: MediaStream } = {};
 let webrtc: MembraneWebRTC | null = null;
 
+export const changeIsTrackEnabled = (clientName: string, sourceType: SourceType, mode: boolean) => {
+  const track = presenterStreams[clientName].getTracks().find((track) => track.kind == sourceType);
```

Comment:
We repeat this fragment so we can move it to the helper function.

Reply:
Fixed